lib/liftoff/cli.rb

```diff
@@ -35,7 +35,7 @@ def global_options
           @options[:strict_prompts] = strict_prompts
         end
 
-        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage') do |list|
+        opts.on('--dependency-managers [NAME(s)]', 'Comma separated list of dependency managers to enable. Available options: cocoapods,carthage,bundler') do |list|
```

Comment:
Line is too long. [164/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.